src/Zone/ZoneController/tracker.d.ts

```diff
@@ -0,0 +1,36 @@
+import Signal from "Zone/Signal";
+
+interface Tracker {
+	trackers: never;
+	itemAdded: Signal<(item: BasePart | Model) => void>;
+	itemRemoved: Signal<(item: BasePart | Model) => void>;
+	bodyPartsToIgnore: {
+		UpperTorso: boolean;
+		LowerTorso: boolean;
+		Torso: boolean;
+		LeftHand: boolean;
+		RightHand: boolean;
+		LeftFoot: boolean;
+		RightFoot: boolean;
+	};
+
+	name: "player" | "item";
+	totalVolume: number;
+	parts: Array<BasePart>;
+	partToItem: Map<BasePart, BasePart | Model>;
+	items: Array<BasePart | Model>;
+	whitelistParams: OverlapParams;
```

Comment:
I think whitelistParams can be undefined as well

https://github.com/christopher-buss/rbxts-zone-plus/blob/48414def2047eced5ffb85870479ec8ae364a2af/src/Zone/ZoneController/Tracker.lua#L69C7-L69C22